### ISSUE TITLE
Release Notes

### ISSUE THREAD

**Issue by pradeepcheers** [IBM/JSONata4Java#161]
Where can I see all the relevant changes associated to the release versions please?

**Comment by wnm3** [comment]
I have been lax about summarizing the changes. I've posted changes against issues people have raised and if the changes solved their problems they closed them.

I'll try to create a change list moving forward.

**Comment by wnm3** [comment]
I am testing against jsonata.org's jsonata.js version 1.8.4 test suite. However, there are about 400 tests we don't currently pass. This was mainly due to a problem with the parsing that I fixed about a week ago. The problem was the entire expression was not being parsed so it was impossible to perform the full interpretation, etc. I fixed this issue last week were we now parse through the entire expression so everything gets put into a parse tree or we get a parse exception. However, I haven't had a chance to begin retesting the areas where we were not compatible so it will take time to add in more capabilities.

I consider it a bug if you can perform something in jsonata.js that is not possible in JSONata4Java. And there are open issues for such problems.

If there is a particular capability you need open an issue and I'll try to address it.

**Comment by tiffmaelite** [comment]
@wnm3 I think it would be very helpful to most users to at least be able to navigate the repository versions using git tags. Would it be feasible for you to push a new git tag every time you change the project version in pom.xml? From there on, we can manually create diffs of 2 versions and easily check whatever we need, such as the jsonata.js underlying version in use.

For example, at the moment, I have difficulties assessing benefits vs. risks of upgrading from 1.7.9 to 1.8.0 and even up to the version currently mentioned in the README, which is 2.5.0.

**Comment by wnm3** [comment]
Yes, sorry I have been lax. Latest change was to move from JDK 1.8 to JDK 11
